src/resources/ha-style.js

```diff
@@ -35,6 +35,7 @@ documentContainer.innerHTML = `<custom-style>
       --secondary-text-color: #727272;
       --text-primary-color: #ffffff;
       --disabled-text-color: #bdbdbd;
+      --dark-text-color: #FFF;
```

Comment:
This should be `--text-dark-color: #FFF`, because it is text for on the dark color. Just like we have `text-primary-color`, which is text for on the primary color.